wurst/assets/LocalAssets.wurst

```diff
@@ -31,10 +31,14 @@ public class LocalUnits
     // A nonexistent model used to hide units.
     static constant dummy = "Dummy.mdl"
 
-public class LocalBuildings
-    static constant forge          = "Models\\Forge.mdx"
-    static constant armory         = "Models\\Armory.mdx"
-    static constant mixingPot      = "Models\\MixingPot.mdl"
+public class LocalModelBuildings
```

Comment:
I would avoid mentioning `Model` to match the standard library assets. Having 2 classes is good though.

Reply:
Thought it would be more explicit this way

Reply:
I think it's clear in context because it's obvious when you're using a model, so just sticking with the standard convention is better.